### ISSUE TITLE
`guess()` works locally but not remotely!

### ISSUE THREAD

**Issue by fjeddy** [moment/moment-timezone#406]
Hello,

I am working on Windows 10 using Bash Ubuntu that comes with Windows 10 by default. I setup and configured moment-timezone, extremely satisfied that everything was working exactly like it should.

or..... ?

I moved the developement to production and realized that moment-timezone's `guess()` suddenly stopped working all together, it would and will always return the servers timezone "America/New York" instead of mine "Europe/Oslo" or "Europe/Paris". The server is running on a linux box with Ubuntu 14.04, the NodeJS app is also running behind a Nginx proxy, not sure if this is relevant in terms of http headers and such ?

I thought moment-timezone based their findings entirely on the user and the browser, so if i move a website from one server to another, but visit it from the exact same computer with the exact same browser, why would the output / result be different?

Encountered this before and know of a possible fix? Not using the browser version at all, just the NodeJS version.

Note: By default, locally, moment-timezone would set the time to GMT, an hour behind me, using `guess()` it set it correctly (read: only locally), i find this behavior odd as well, considering that moment-timezone should by default use the servers timezone and working locally that would mean my timezone? Feels like all theses "issues" are on my end.

**Comment by mj1856** [comment]
Probably you don't have data loaded for moment-timezone.  It will only guess from the zones it has loaded.

Check the debug console in your browser where it fails and see if there's an error attempting to load the data.

**Comment by howdycrystalm** [comment]
@fjeddy did you ever find a solution to this? I'm having the exact same issue where `guess()` works perfectly locally, but once pushed to my server, timezone is based off the server's timezone.

**Comment by howdycrystalm** [comment]
@maggiepint Thanks for the reply!  I've used `npm install moment-timezone`, which gave me the impression that all data was pre-loaded.  Should I also download moment-timezone.min.js?

Here's a little more info about what I'm trying to accomplish.  In short,  I'd like to capture the date and time a user adds data to the database based on their local time. 

Here's what I've tried that at least _seems_ to work on my local machine, but fails on my server when friends in other timezones test out my site. By fail I mean shows my server's time, PST, instead of user's local time.

`var moment = require('moment-timezone);

var guessUserTimeZone = moment.tz.guess();
var date = moment().tz(guessUserTimeZone).format('MMMM Do YYYY, h:mm:ss a');'
